doc/deployment_guide.md

```diff
@@ -127,7 +127,7 @@ The network consists of the following major network infrastructure layouts:
 * The following features are added in the JetPack Automation and the Ready Architecture for RedHat OpenStack:*
 
 * Support for the latest release of Red Hat OpenStack Platform 13 including the latest updates.
-* Support for Dell EMC PowerEdge R640 and R740/740xd servers with 2nd Generation Intel® Xeon®Scalable Processors (Cascade Lake) and DDR4 2933MHz memory DIMMs
+* Support for Dell EMC PowerEdge R640 and R740/740xd servers with second Generation Intel® Xeon®Scalable Processors (Cascade Lake) and DDR4 2933MHz memory DIMMs
```

Comment:
Leave it at 2nd since that is what we have in the RA as well as Marketing collateral.  This is the term we got from Intel btw.